### PR TITLE
chore: add english version of the demo

### PR DESCRIPTION
**Pull request by ncclementi**

### CODE REVIEW

File: demo/ibis-demo.en.qmd [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# Ibis Dempo
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
# Ibis Demo
```

File: demo/ibis-demo.en.qmd [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+basics = con.read_parquet("imdb_title_basics.parquet", table_name="imdb_title_basics")
+```
+
+We can list the names of the tables
+
+```{python}
+con.list_tables()
+```
+
+And render an abstract version of the expresion withe the column's names and types:
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
And render an abstract version of the expression with the column's names and types:
```

File: demo/ibis-demo.en.qmd [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+For our final analysis, we will clean a few things.
+
+We will select only the titles with the "movie" type and discard the adult movies.
+
+To do this, we will use a list of predicates
+
+```{python}
+basics = (
+    basics
+    .filter([basics.title_type == "movie", basics.is_adult == 0])
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
    .filter(basics.title_type == "movie", basics.is_adult == 0)
```

Not a necessary change, but `filter` accepts variable args now, so it doesn't need to be a list of predicates

File: demo/ibis-demo.en.qmd [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+```{python}
+basics
+```
+
+Then, we join the `basics` table and the `ratings` table using the `tconst` column, 
+and we execute
+
+```{python}
+basics.join(ratings, "tconst").to_pandas(limit=10)
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
Then, we join the `basics` table and the `ratings` table using the `tconst` column.

```{python}
basics.join(ratings, "tconst")
```

I think we're in interactive mode at this point, yeah?

File: demo/ibis-demo.en.qmd [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+```
+
+We order by `average_rating`
+
+```{python}
+basics.join(ratings, "tconst").order_by(_.average_rating.desc())
+```
+
+And filter the titles with more than one million votes
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
And keep only those titles with more than one million votes
```

File: demo/ibis-demo.en.qmd [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+
+```{python}
+(
+    basics.join(ratings, "tconst")
+    .order_by(_.average_rating.desc())
+    .filter(_.num_votes > 1e6)
+)
+```
+
+## Table joins (`join`)
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd:195]: This seems... out of place?

File: demo/ibis-demo.en.qmd [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+pgcon.list_tables()
+```
+
+Execute the same expresion in Postgres:
+
+```{python}
+pgcon.to_pandas(topfilms)
+```
+
+¡That's it, that's Ibis!
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
That's it, that's Ibis!
```

File: demo/ibis-demo.en.qmd [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+pgcon = ibis.postgres.connect()
+```
+
+### Check that the tables exist
+
+```{python}
+pgcon.list_tables()
+```
+
+Execute the same expresion in Postgres:
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
Execute the same expression in Postgres:
```

File: demo/ibis-demo.en.qmd [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+
+```{python}
+pgcon.to_pandas(topfilms)
+```
+
+¡That's it, that's Ibis!
+
+Ibis has 20+ backends, like Snowflake, DuckDB (what we saw here), and more.
```
Comment (gforsyth) [inline demo/ibis-demo.en.qmd]:
```suggestion
Ibis has 20+ backends, like Snowflake, DuckDB (as seen above), and more.
```

**Review by gforsyth — COMMENTED** [review]
A few small typos, looks good overall!

File: demo/ibis-demo.en.qmd (hunk 6 above)
Comment (ncclementi) [reply demo/ibis-demo.en.qmd:195]: yes, but it was on the original demo so I'll leave it for now, we can revisit the demo content in a follow up

**Review by ncclementi — COMMENTED** [review]
Thanks for the review. Added the comments, I guess I need to review my spellcheck extension. I pushed the notebook with the changes too, all should be updated.

**Pull request merged**